client/webserver/site/src/html/forms.tmpl

```diff
@@ -203,6 +203,31 @@
 </div>
 {{end}}
 
+{{define "changeAppPWForm"}}
+<div class="bg2 px-2 py-1 text-center position-relative fs18">
+  Change Application Password
+  <div class="form-closer hoverbg"><span class="ico-cross"></span></div>
+</div>
+<div class="p-4">
+  <div>
+    <label for="appPW" class="pl-1 mb-1">Current Password</label>
+    <input type="password" class="form-control select" id="appPW" autocomplete="current-password">
+  </div>
+  <div class="mt-2">
+    <label for="newAppPW" class="pl-1 mb-1">New Password</label>
+    <input type="password" class="form-control select" id="newAppPW" autocomplete="new-password">
+  </div>
+  <div class="mt-2">
+    <label for="confirmNewPW" class="pl-1 mb-1">Confirm New Password</label>
+    <input type="password" class="form-control select" id="confirmNewPW" autocomplete="new-password">
+  </div>
+  <div class="d-flex justify-content-end mt-4">
+    <button id="submitNewPW" type="submit" class="col-8 justify-content-center fs15 bg2 selected">Submit</button>
+  </div>
```

Comment:
Would you add some margin between the buttons while you're here?
![image](https://user-images.githubusercontent.com/6109680/110633190-15f9a500-816e-11eb-91ca-7177233a5aa2.png)